### PR TITLE
Add check_copyright.sh script

### PR DESCRIPTION
**Pull request by nurmukhametov**
To use run with any git ref as argument, e.g.:
```bash
$ ./check_copyright.sh HEAD^
```

### CODE REVIEW

**Comment by aneshlya** [comment]
It would be good to add a Github Action job running this script on PR.

**Comment by dbabokin** [comment]
If I understand it correctly, it checks only the last commit in the PR. Any ideas how to check the whole PR?

**Comment by dbabokin** [comment]
> If I understand it correctly, it checks only the last commit in the PR. Any ideas how to check the whole PR?

Oh, looks like I misunderstood it. So does it check the whole PR?

**Comment by nurmukhametov** [comment]
> > If I understand it correctly, it checks only the last commit in the PR. Any ideas how to check the whole PR?
> 
> Oh, looks like I misunderstood it. So does it check the whole PR?

Yeah, it should check the whole PR diffing current state with GITHUB_BASE_REF

**Review by aneshlya — APPROVED** [review]

File: check_copyright.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# ##################################################
+#  Copyright (c) 2022, Intel Corporation
+#  All rights reserved.
+#
+#  Redistribution and use in source and binary forms, with or without
```
Comment (dbabokin) [inline check_copyright.sh:6]: I suggest using SDPX version of the license, like here: https://github.com/ispc/ispc/blob/0387b0679f2951caeff2a32446790980ac92215b/.github/workflows/rebuild-llvm15.yml

Comment (nurmukhametov) [reply check_copyright.sh]: fixed

File: .github/workflows/copyright-check.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2022 Intel Corporation
```
Comment (dbabokin) [inline .github/workflows/copyright-check.yml:1]:
```suggestion
# Copyright 2022, Intel Corporation
```

File: check_copyright.sh [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# Copyright 2022 Intel Corporation
```
Comment (dbabokin) [inline check_copyright.sh:2]:
```suggestion
# Copyright 2022, Intel Corporation
```

**Review by dbabokin — APPROVED** [review]
Changes looks good to me! Thanks for implementing this check.

As a general note - going forward we should probably move this and other scripts to `scripts` folder.

**Pull request merged**